packages/import-bundle/src/index.js

```diff
@@ -78,6 +78,56 @@ export async function importBundle(bundle, options = {}, powers = {}) {
     return namespace;
   }
 
+  // The 'test' format is not generated by bundleSource and is not
+  // serializable as JSON and is not passable because copy-records cannot have symbol keys.
+  if (moduleFormat === 'test') {
+    const exports = bundle[Symbol.for('exports')];
+    if (exports === undefined) {
+      throw new Error(
+        'Cannot import bundle with moduleFormat "test" that lacks an symbol-named property @exports and has likely been partially transported via JSON or eventual-send',
+      );
+    }
+
+    // We emulate a module exports namespace object, which has certain invariants:
+    // Property names are only strings, so we will ignore symbol-named properties.
+    // All properties are enumerable, so we will ignore non-enumerable properties.
+    // All properties are non-writable values.
```

Comment:
Nit: String keys on Module namespace objects are actually all `writable` but we purposely deviate here.